src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/slicing/CountrySlicingProcessor.java

```diff
@@ -341,6 +345,13 @@ else if (piece1.getStartNode().equals(piece2.getEndNode()))
         return results;
     }
 
+    private boolean canSkipSlicingIfSingleCountry(final Way way)
```

Comment:
Nit: maybe a quick comment saying we can skip slicing for a single country unless the way is a Pier or a Ferry, since both of these features can extend into the surrounding water.